batch/provision.yml

```diff
@@ -5,14 +5,13 @@
   gather_facts: False
   vars:
     # The following vars should probably be overridden
-    - name: ETL
+    - name: Lagunita 4.7.2_6 EMR Cluster
```

Comment:
Yes, there doesn't seem to be a way of deleting terminated clusters otherwise I wouldn't have used this.